src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2102,6 +2126,43 @@ public List<CertificateDTO> getEntitlementCertificates(@Verify(Consumer.class) S
             .collect(Collectors.toList());
     }
 
+    /**
+     * Retrieves a {@link AnonymousContentAccessCertificate} for the anonymous cloud consumer. If there
+     * if no existing certificate, a new one will be created.
+     *
+     * @param consumer
+     *  the anonymous cloud consumer to retrieve a certificate for
+     *
+     * @param serials
+     *  certificate serials used to filter out a certificate
+     *
+     * @throws IseException
+     *  if unable to retrieve or create an {@link AnonymousContentAccessCertificate}
+     *
+     * @return a {@link AnonymousContentAccessCertificate}, or null if a certificate was not able to be
+     *  retrieved, created, or filtered by the serial IDs
+     */
+    private CertificateDTO getCertForAnonCloudConsumer(AnonymousCloudConsumer consumer, String serials) {
+        Certificate caCert;
+        try {
+            caCert = this.contentAccessManager.getCertificate(consumer);
+        }
+        catch (Exception e) {
+            String msg = "Unable to retrieve or create anonymous content access certificate for consumer";
```

Comment:
Updated

Reply:
The way this was translated (`i18n.tr(msg)`) won't work. This method (`tr`) needs to get the raw string itself  `i18n.tr("Unable to..")` instead of a variable, because that's what the gettext cli looks for when extracting translatable strings.

Reply:
Oh interesting. Updated